File: JAVA/Learning_0/JAVA-100K/BiggerOfThreeNumbers.java
import java.util.Scanner;

public class BiggerOfThreeNumbers {
    public static void main (String ar[]) {
        Scanner sc = new Scanner(System.in);
        //Main code

        System.out.println("Enter three numbers:");
        int num1 = sc.nextInt();
        int num2 = sc.nextInt();
        int num3 = sc.nextInt();
        if (num1>num2) {
            if (num1>num3) {
                System.out.println(num1+" is greatest");
            } else {
                System.out.println(num3 + " is greatest");
            }
        } else {
            if (num2>num3) {
                System.out.println(num2+" is greatest");
            } else {
                System.out.println(num3+" is greatest");
            }
        }

        sc.close();

    }
    
}

File: C/Learn/9BIGGER-OF-THREE-NUMBERS/src/BIGGER-OF-THREE-NUMBERS.c
/*
 ============================================================================
 Name        : BIGGER-OF-THREE-NUMBERS.c
 Author      : 
 Version     :
 Copyright   : Your copyright notice
 Description : Hello World in C, Ansi-style
 ============================================================================
 */

#include <stdio.h>
#include <stdlib.h>

int main(void) {
	int num1, num2, num3;
	setbuf(stdout, NULL);
	printf("Enter the three numbers to be compared : ");
	scanf("%d%d%d", &num1, &num2, &num3);
	if (num1>num2) {
		if (num1>num3) {
			printf("%d is greatest", num1);
		} else {
			printf("%d is greatest", num3);
		}
	} else {
		if (num2>num3) {
			printf("%d is greatest", num2);
		} else {
			printf("%d is greatest", num3);
		}
	}
	return EXIT_SUCCESS;
}
